README.md

```diff
@@ -45,12 +45,22 @@ The function `expectMessage(value, message, style = {})` takes three arguments:
 
 #### Example
 
+With pre-defined style:
 ```js
 test("simple test", () => {
-  expectMessage(1 + 1, "This is a level 3 message", 3).toBe(11);
+  expectMessage(1 + 1, "This is an error message", style="error").toBe(11);
 });
 
-// This test will fail and print "This is a level 3 message" in yellow with a red background.
+// This test will fail and print "This is an error message" in yellow with a red background.
+```
+
+With your own style:
+```js
+test("simple test", () => {
+  expectMessage(1 + 1, "This is my own error message", style={fg: "magenta", bg: "cyan", effects: "reverse"}).toBe(11);
```

Comment:
same question here

Reply:
Same answer, it works but we can just remove "style"